pycbc/workflow/coincidence.py

```diff
@@ -276,9 +276,11 @@ class PyCBCMultiifoAddStatmap(PyCBCMultiifoCombineStatmap):
 
     current_retention_level = Executable.MERGED_TRIGGERS
     def create_node(self, statmap_files, background_files, tags=None):
+        if tags is None:
+            tags = []
```

Comment:
We have warnings against a lack of tags elsewhere - probably should have the same here?

Reply:
So no tags is okay in principle and probably fine for this case, which may only be called once. What cases do we show warnings when no tags are given?

Reply:
There is a warning for empty tags on:
https://github.com/gwastro/pycbc/blob/master/pycbc/workflow/core.py#L514

That is probably as it is a file rather than an executable though

Reply:
Ah, no. That's not a warning that there's *no* tags, but a warning that one of the tags that was sent was `""` (an empty string). This should be an error, so any codes seeing this warning need fixing!

Reply:
Ah makes sense. Ignore my brain fart